### PR TITLE
Switch to NPM as default client package manager

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Related to jhipster/generator-jhipster#8162

### CODE REVIEW

**Comment by deepu105** [comment]
Yup

**Comment by deepu105** [comment]
Technically its not required switch sub projects like these but yes lets bee consistent for our own sake

File: README.md [hunk 1]
```diff
@@ -35,21 +35,22 @@ Go to the [ng-jhipster project](https://github.com/jhipster/ng-jhipster) and cli
...
+### Set NPM to use the cloned project
 
-Go to your generated JHipster application and run...
+In your cloned `ng-jhipster` project, type `npm link`.
 
-    git checkout -- yarn.lock
-    yarn cache clean ng-jhipster
-    rm -rf `yarn cache dir`/.tmp
-    yarn add path/to/ng-jhipster/ng-jhipster-vX.Y.Z.tgz
+This will do a symbolic link from the global `node_modules` version to point to this folder.
```
Comment (deepu105) [inline README.md]: Did you check this does it work? I guess I was using the pack command with NPM as well since the linking part was not working with AOT

Comment (pascalgrimaud) [reply README.md]: I use the same README than React, but to be honest, I didn't test this

**Comment by deepu105** [comment]
For React we dont have the problem, only for Angular due to AOT

Thanks & Regards,
Deepu

**Comment by pascalgrimaud** [comment]
Ok @deepu105 : I will update my PR to use packaging instead of npm link

**Comment by pascalgrimaud** [comment]
completely forgot this one. I will try to update it today

**Comment by pascalgrimaud** [comment]
done: updated the README to use `npm pack` and the `package-lock.json` too

**Pull request merged**